tests/resources/test_branch.py

```diff
@@ -426,16 +563,277 @@ def test_get_by_root_id(session, collection, branch_path):
     )]
 
 
-def test_get_by_root_id_not_found(session, collection, branch_path):
+def test_get_by_root_id_not_found_deprecated(session, collection, branch_path):
     # Given
     session.set_response({'response': []})
     root_id = uuid.uuid4()
 
     # When
-    with pytest.raises(NotFound) as exc:
-        collection.get_by_root_id(branch_root_id=root_id)
+    with pytest.deprecated_call():
+        with pytest.raises(NotFound) as exc:
+            collection.get_by_root_id(branch_root_id=root_id)
 
     # Then
     assert str(root_id) in str(exc)
     assert "branch root" in str(exc).lower()
     assert "latest" in str(exc).lower()
```

Comment:
```suggestion
    assert LATEST_VER.lower() in str(exc).lower()
```

Reply:
This would be bad practice. It would mean if we screwed up the string in `branch.LATEST_VER`, the tests wouldn't catch it.

That being said, it's not the worst idea to define an independent constant in the test file.